doc/nrf/libraries/dfu/suit_dfu.rst

```diff
@@ -0,0 +1,50 @@
+.. _subsys_suit:
+
+Software Updates for Internet of Things (SUIT)
+##############################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Software Updates for Internet of Things (SUIT) DFU library provides functionality for a Local domain Core to orchestrate the update based on the SUIT manifest.
+
+Overview
+********
+
+The SUIT DFU library provides APIs to start as well as prepare the device before the update procedure is started.
+The preparation for the DFU may be described as a sequence inside a SUIT manifest (suit-payload-fetch).
+To interpret and execute this sequence, a SUIT processor must be instantiated inside the Local domain Core firmware.
```

Comment:
```suggestion
To interpret and execute this sequence, a SUIT processor must be instantiated inside the firmware of the local domain core.
```

Reply:
Can you define the difference between core and domain?

Reply:
I think there are official definitions for nRF54. Do you know where they can be found?
Unfortunately, my understanding is not the one to use in this case :slightly_smiling_face:

Reply:
From the definitions I heard about, they should be synonyms. @hubertmis is this the case? If that's the case, I'd remove the word core and just leave domain

Reply:
Depending on the context, sometimes they are synonyms. I agree it would be clearer tot drop "domain"